src/en/controllers-creating.md

```diff
@@ -7,20 +7,34 @@ TODO: Review again soon (created: March 2016)
 Use the `juju bootstrap` command to create a controller (and model) for a given
 cloud provider:
 
-`juju bootstrap <controller name> <provider name> [--options]`
+`juju bootstrap [options] <controller name> <provider name>`
 
 See `juju help bootstrap` for details on this command or see the
 [command reference page](./commands.html#juju-bootstrap).
 
 
+## Notes
+
+The '--upload-tools' option is used to make agent software available that is
+more recent than the default binary. This is done when some features may not
+yet be compiled in to the agent for the Ubuntu release being installed. Note
+that Juju will default to the latest LTS (see `distro-info --lts` command).
+
+
 ## Examples
 
-**1.** The controller is for Rackspace. We call the controller 'controller-rackspace'
-and use `--upload-tools` to make the agent software available to our default
-series (specified in the provider's configuration):
+**1.** Create Rackspace controller 'controller-rackspace':
+
+```bash
+juju bootstrap \
+	--debug --upload-tools --config=~/config-rackspace.yaml \
+	controller-rackspace rackspace
+`
+
+**2.** Create LXD controller 'lxd-trusty':
 
 ```bash
 juju bootstrap \
-	controller-rackspace rackspace \
-	--debug --upload-tools --config=~/config-rackspace.yaml
+	--debug --upload-tools \
+	lxd-trusty lxd
 ```
```

Comment:
only if something goes very wrong. The output without debug is extensive enough for most users. I just think we shouldn't complicate things with non-essential options